### DISCUSSION TITLE
Were Districts removed from region drop down?

### DISCUSSION THREAD

**Discussion by OSAwebsite** [code4recovery/12-step-meeting-list#856] [Q&A]
### Contact Details

os.website.a86@gmail.com

### Website With Issue

www.osatest.al-anon.alateen.on.ca

### What happened?

I think I have mentioned this before but I perceive this as a bug: where did the district option go in the region drop down?
I know districts are an administrative element of how meetings are spread over the area. But existing members use them to double check all meetings in their district region. Is this coming back? Who agrees with me that this is a useful feature to hold on to?

I have not installed the latest version yet but I am running on Wordpress 6 on my staging site. The site is under construction but I am sure you don't need to access it to identify this lost feature.

### TSML Version

Other (Identify version in text above)

### Wordpress Version

Latest (default)

**Comment by OSAwebsite** [comment]
I think I forgot about this: the removed districts only are an issue in the new design. Senior moment. Nevertheless: can you give an update about the new design? Are the districts coming back? Sorry if this caused confusion...

**Reply by tech2serve** [reply]
Regrettably, I'm currently fighting my computer trying to get a Wordpress dev site working again. 

By "new design", do you mean TSML-UI?

**Reply by tech2serve** [reply]
Ok, after getting a new site up and running, I do see your concern. It seems like a reasonable request, but I'm not sure if it is a bug or is a feature @joshreisner hasn't implemented in TSML-UI.

After he has a chance to look at this, we'll discuss opening an issue with the TSML-UI repo.

Thanks for bringing it up!

**Reply by joshreisner** [reply]
no, it's a conscious decision not to include "switch to districts" functionality in TSML UI. please see https://github.com/code4recovery/tsml-ui/issues/125 and https://github.com/code4recovery/12-step-meeting-list/discussions/762#discussioncomment-2568747

you can still search for districts using keyword search

**Comment by OSAwebsite** [comment]
Yes, I had forgotten that I toggled on the option to see the new UI. Sorry
to hear about computer troubles. This too will pass :)


-
Yours in service,
Nicky D
Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*

**Comment by OSAwebsite** [comment]
I remember the dialogue but I don’t agree, mostly because the plug-in still
does not give people an intuitive alternative to select all meetings in the
geographical area and seeing them in a printable pdf list or download.

The keyword search is still confusing to many. Removing the district search
option from the most intuitive spot narrows the options and I don’t think
that is helpful, in particular for newcomers in need.

District meeting lists are in big demand at my end and the website (Ontario
South Al-Anon) therefore offers them separately. The alternative is having
every district create their own sheets and lists that are outdated the
moment they are printed. Again: more confusing than helpful.

To me it therefore makes a lot of sense to incorporate this in TSML. Not
because we are including an administrative element that supposedly means
nothing to newcomers. But to offer the option of quickly checking a region
in the true sense of that word.

What is called a region in TSML really only is a list of towns and cities.
To look for meetings in several of these takes a lot of user actions if you
know what I mean. It is so much quicker and easier to look for districts
instead. Even if the term has no initial meaning to a newcomer, it becomes
clear quickly when they try it out and see that a district is a collection
of meetings in a logical geographical area. It also takes away the need to
know where all towns and cities in the region list are located. Maybe that
is more important to people like me who do not live where they grew up.
Canadians might know their surroundings better.

So I request once more to have this removed feature be reconsidered in the
new UI. Let’s ask the memberships about this before we take away something
that truly has meaning and use to many of us.  Not ‘us’ as those who are in
the know about the plug-in. But ‘us’ as those who are still out there
suffering and searching for meetings, often in a state of chaos and panic.

Thanks!

On Tue, Jul 5, 2022 at 6:11 PM Josh Reisner ***@***.***>
wrote:

> no, it's a conscious decision not to include "switch to districts"
> functionality in TSML UI. please see code4recovery/tsml-ui#125
> <https://github.com/code4recovery/tsml-ui/issues/125>
>
> you can still search for districts using keyword search
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/856#discussioncomment-3088094>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOIWBF56ZZE6YXFLB36HGXDVSSXHPANCNFSM52WXKWVQ>
> .
> You are receiving this because you authored the thread.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/856/comments/3088094@
> github.com>
>
-- 


-
Yours in service,
Nicky D
Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*

**Reply by Grateful-Viv** [reply]
You can search by district number. Does that help?
